packages/flutter_tools/lib/src/build_system/targets/localizations.dart

```diff
@@ -36,11 +36,14 @@ Future<void> generateLocalizations({
     'bin',
     'gen_l10n.dart',
   );
+  final String projectDirPath = projectDir.path.replaceAll("'", '');
```

Comment:
You shouldn't need to manipulate the path string like this - what problem is this solving?

Reply:
This was a mistake I made. The test failed for some other reason and I couldn't understand why. I'll remove this line in a subsequent commit. Thanks for the catch